creator-node/src/utils/validateAudiusUserMetadata.js

```diff
@@ -0,0 +1,34 @@
+const Web3 = require('web3')
+const web3 = new Web3()
+
+/**
+ * Validates that the user id was signed by the associated wallets
+ */
+const validateAssociatedWallets = (metadataJSON) => {
+  if ('user_id' in metadataJSON && 'associated_wallets' in metadataJSON) {
+    const userId = metadataJSON['user_id']
+    const walletMappings = metadataJSON['associated_wallets']
+    if (typeof walletMappings !== 'object') return true
```

Comment:
if it's not an object, should we return false?
what if you wrote associated_wallets = []

Reply:
yeah, ideally we would use the same libs JSON validate function actually. I figured that either this error on incorrect format and the user metadata is not saved, or it is valid and DP can index it.

Reply:
fyi -- needed to make a change here. typeof null = object lol